packages/opentelemetry-plugin-user-interaction/test/userInteraction.nozone.test.ts

```diff
@@ -0,0 +1,300 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// because of zone original timeout needs to be patched to be able to run
+// code outside zone.js. This needs to be done before all
+const originalSetTimeout = window.setTimeout;
+
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { isWrapped, LogLevel } from '@opentelemetry/core';
+import * as tracing from '@opentelemetry/tracing';
+import { WebTracer } from '@opentelemetry/web';
+import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
+import { UserInteractionPlugin } from '../src';
+
+import {
+  assertClickSpan,
+  DummySpanExporter,
+  fakeInteraction,
+  getData,
+} from './helper.test';
+
+const FILE_URL =
+  'https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/master/package.json';
+
+describe('UserInteractionPlugin', () => {
+  describe('when zone.js is NOT available', () => {
+    let userInteractionPlugin: UserInteractionPlugin;
+    let sandbox: sinon.SinonSandbox;
+    let webTracer: WebTracer;
+    let dummySpanExporter: DummySpanExporter;
+    let exportSpy: any;
+    let requests: any[] = [];
+    beforeEach(() => {
+      sandbox = sinon.createSandbox();
+      const fakeXhr = sandbox.useFakeXMLHttpRequest();
+      fakeXhr.onCreate = function(xhr: any) {
+        requests.push(xhr);
+        setTimeout(() => {
+          requests[requests.length - 1].respond(
+            200,
+            { 'Content-Type': 'application/json' },
+            '{"foo":"bar"}'
+          );
+        });
+      };
+
+      sandbox.useFakeTimers();
+
+      userInteractionPlugin = new UserInteractionPlugin();
+
+      sinon
+        .stub(userInteractionPlugin, 'getZoneWithPrototype')
+        .callsFake(() => undefined);
+
+      webTracer = new WebTracer({
+        logLevel: LogLevel.ERROR,
+        plugins: [userInteractionPlugin, new XMLHttpRequestPlugin()],
+      });
+
+      dummySpanExporter = new DummySpanExporter();
+      exportSpy = sandbox.stub(dummySpanExporter, 'export');
+      webTracer.addSpanProcessor(
+        new tracing.SimpleSpanProcessor(dummySpanExporter)
+      );
+
+      // this is needed as window is treated as scope and karma is adding
+      // context which is then detected as spanContext
```

Comment:
We might fix this differently by removing window from scope managers when active span doesn't exist I think this might be probably a proper fix, I need to check it but the fix will come in different PR anyway, for now I would say it will stay as it is